src/Network/Minio/Data.hs

```diff
@@ -184,21 +184,32 @@ type Object = Text
 -- TODO: This could be a Sum Type with all defined regions for AWS.
 type Region = Text
 
--- | A type alias to represent an Entity-Tag returned by S3-compatible
--- APIs.
+-- | A type alias to represent an Entity-Tag returned by S3-compatible APIs.
 type ETag = Text
 
 -- |
 -- Data type represents various options specified for PutObject call.
 -- To specify PutObject options use the poo* accessors.
 data PutObjectOptions = PutObjectOptions {
+  -- | Set a standard MIME type describing the format of the object.
     pooContentType        :: Maybe Text
+  -- | Set what content encodings have been applied to the object and thus
+  -- what decoding mechanisms must be applied to obtain the media-type
+  -- referenced by the Content-Type header field.
   , pooContentEncoding    :: Maybe Text
+  -- | Set presentational information for the object.
   , pooContentDisposition :: Maybe Text
+  -- | Set to specify caching behavior for the object along the
+  -- request/reply chain.
   , pooCacheControl       :: Maybe Text
+  -- | Set to describe the language(s) intended for the audience.
   , pooContentLanguage    :: Maybe Text
+  -- | Set to possible values are Standard and ReducedRedundancy. The
```

Comment:
This should be "Set Storage class. Possible values are Standard and ReducedRedundancy ..."